src/pages/LoadingPage.tsx

```diff
@@ -1,3 +1,70 @@
-const LoadingPage: React.FC = () => <div>I am the loading page</div>;
+import { useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { LogTypes } from "constants/enums";
+import {
+  getEvergreenTaskLogURL,
+  getEvergreenTestLogURL,
+  getResmokeLogURL,
+} from "constants/logURLTemplates";
+import { slugs } from "constants/routes";
+import { useLogContext } from "context/LogContext";
+import { useAxiosGet } from "hooks/useAxios";
```

Comment:
I think we might want to import this `from "hooks"` (unless you want to get rid of the index file in `/hooks`)